client/components/LearnerViewDirective.js

```diff
@@ -733,6 +739,32 @@ tie.directive('learnerView', [function() {
         };
 
         /**
+          * Highlights syntax errors in the coding UI
+          *
+          * @param {number} lineNumber
+          */
+        var highlightLine = function(lineNumber) {
+          var actualLineNumber = lineNumber - 1;
+          var codeLines = document.querySelectorAll('.CodeMirror-line');
+          if ($scope.isInDarkMode) {
+            codeLines[actualLineNumber].classList.add('tie-syntax-error-line',
+              'night-mode');
```

Comment:
(Oh, sorry, my bad. I just realized you hadn't actually addressed that comment yet.)